apps/platform/src/components/ui/loading.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Loading Component
+ * 
+ * This component creates a skeleton representation of a UI structure,
+ * followed by rendering actual data after loading. It utilizes utility
+ * functions (`cn` for classNames).
+ * Uses Skeleton component from  '@/components/ui/skeleton'
+ * Props:
+ * - className: Additional classNames to apply to the main container.
+ * 
+ * Features:
+ * - Generates 3 rows of skeleton content to mimic the "Key Manager" layout.
+ * 
+ * Uses:
+ * ```
+ * @/lib/utils - to create the structure of skeleton.
+ * ```
+ * Author: Nirajan1-droid
+ * Date: 2024-07-08
+ */
+import * as React from 'react';
+import { cn } from '@/lib/utils';  
+import { Skeleton } from '@/components/ui/skeleton'; 
+
+const Loading = React.forwardRef<HTMLDivElement>(() => {

```

Comment:
We are not using ref in this component, so forwardRef is not needed in this component. Convert it in a regular react function, not an arrow function.